### ISSUE TITLE
[styletron-utils] injectStylePrefixed repeats atomic classes in returned class string for some vendor-prefixed declarations

### ISSUE THREAD

**Issue by muratgozel** [rtsao/styletron#42]
Hi, today i was testing styletron for the first time in my sample react app. I realize that it injects some class names more than once on both client and server side.

Here is a part of component file:
```javascript
const Button = styled('a', () => ({
  display: 'flex',
  alignItems: 'center',
  fontSize: '1.2rem',
  lineHeight: '1',
  padding: '0.6rem 2.4rem',
  borderRadius: '8px',
  fontWeight: 'normal',
  textDecoration: 'none',
  transition: 'all 0.5s ease',
  backgroundColor: app.branding.colors[0],
  color: 'white',
  ':hover': {
    color: 'black'
  }
}))

class Header extends Component {
  render() {
    const {clientFeatures, language} = this.props

    return (
      <header>
        Test Header
        <Button href='https://google.com'>Test Styletron</Button>
      </header>
    )
  }
}
```

Stylesheet generated with no problem:
```html
<style class="_styletron_hydrate_">.a{display:-webkit-box}.b{display:-moz-box}.c{display:-ms-flexbox}.d{display:-webkit-flex}.e{display:flex}.f{-webkit-align-items:center}.g{-ms-flex-align:center}.h{-webkit-box-align:center}.i{align-items:center}.j{font-size:1.2rem}.k{line-height:1}.l{padding:0.6rem 2.4rem}.m{border-radius:8px}.n{font-weight:normal}.o{text-decoration:none}.p{-webkit-transition:all 0.5s ease}.q{transition:all 0.5s ease}.r{background-color:#E63020}.s{color:white}</style>
```

But class names in html element contains some classes more than once: (like e, p and q)
```html
<a href="https://google.com" class=" a b c d e e f g h i j k l m n o p p p q q r s" data-reactid="7">Test Styletron</a>
```

This is not look right. Can you help with this?

**Comment by rtsao** [comment]
It's not actually injecting twice (this is memoized by styletron-core) but because of the way the vendor prefixer works it is being repeated in the returned class string. Working on a simple fix that will mitigate this.

**Comment by rtsao** [comment]
Released in v2.2.1
